Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/MongoNamespace.swift

```diff
@@ -2,31 +2,22 @@
  * A MongoDB namespace, which includes a database name and collection name.
  */
 public struct MongoNamespace: Codable, CustomStringConvertible, Hashable {
+    private enum CodingKeys: String, CodingKey {
```

Comment:
[minor] so I'm assuming this works, but then the underlying data structure for synchronization configs will be different on Android and iOS. I think it's important that we somehow maintain the "db.coll" String representation of the namespace.

Reply:
Problem is, that's not how namespaces are represented by our API. I feel that API parity is more important.

Reply:
where do we get a namespace back from our API?

Reply:
ChangeEvents, unless the docs are wrong

Reply:
ah it looks like in Android we decode them manuall in the change event decoder:

https://github.com/mongodb/stitch-android-sdk/blob/master/core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/ChangeEvent.java#L368

Reply:
I think keeping data the same on the device is going to be important as well, especially if we're going to end up going with a common C++ core. It helps us avoid doing local device data migrations in the future

Reply:
So I think we should still encode the MongoNamespace as `"\(db).\(coll)"` in the sync config